cvat-data/webpack.config.js

```diff
@@ -50,20 +44,89 @@ const webConfig = {
     },
 };
 
-const nodeConfig = {
-    target: 'node',
-    mode: 'development',
-    devtool: 'source-map',
-    entry: './src/js/cvat-data.js',
+
+const worker_img = {
+    target: 'web',
+    mode: 'production',
+    entry: './src/js/unzip_imgs.js',
     output: {
         path: path.resolve(__dirname, 'dist'),
-        filename: 'cvat-data.node.js',
-        library: 'cvatData',
-        libraryTarget: 'commonjs',
+        filename: 'unzip_imgs.js',      
     },
-    stats: {
-        warnings: false,
+    devServer: {
```

Comment:
Do you actually need devServer in this configuration?

Reply:
I don't know. If I'll delete this code will page work?